src/test/resources/settings.json

```diff
@@ -16,7 +16,7 @@
         "profile.default_content_settings.popups": "0",
         "disable-popup-blocking": "true",
         "download.prompt_for_download": "false",
-        "download.default_directory": "//home//selenium//downloads"
+        "download.default_directory": ".//home//selenium//downloads"
```

Comment:
oh no, why? it is not a local path but global one

Reply:
fixed